### DISCUSSION TITLE
assembly instructions

### DISCUSSION THREAD

**Discussion by myreal84** [WhereslyClock/MyWhereslyClock#10] [Q&A]
Hi, i just found your version of the Weasley Clock and i totally loved it but i'm not 100% sure about the assembly of the clockwork

there a just 3 steppergear files ? why not just 1? or 4? :) 
Where did you put the washer?

**Comment by WhereslyClock** [comment]
Hi there are four steppers one per hand, the washer goes below all the
gears on the central spindle else the lowest gear has friction against the
base so it just lessens it.
if you reply to ***@***.***  with your email address I can share the
fusion 360 files with you soyou can see the whole construction

**Reply by WhereslyClock** [reply]
hit me up on bluesky, binaryjam . tocker .co .uk   if you didnt get a direct mail with email
